.github/workflows/build.yml

```diff
@@ -0,0 +1,21 @@
+name: Build
+on:
+  pull_request:
+    paths-ignore:
+    - '**.go'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v1
+
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.13
+
+    - name: build rook
+      run: |
+        # set VERSION to a dummy value since Jenkins normally sets it for us. Do this to make Helm happy and not fail with "Error: Invalid Semantic Version"
```

Comment:
Helm doesn't complain if I build without the VERSION set. When you're building locally do you have to set it as well?

Reply:
It seems to be happening here only...